tests/v2/validation/charts/monitoring.go

```diff
@@ -4,12 +4,13 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/prometheus/common/model"
 	"net/url"
 	"time"
 
 	"github.com/pkg/errors"
 	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
-	"github.com/rancher/rancher/pkg/controllers/managementuserlegacy/alert/config"
+	alertconfig "github.com/rancher/rancher/pkg/controllers/managementuserlegacy/alert/config"
```

Comment:
Here, I just added an alias to the import.

Reply:
I might need more context about this; at first, we were seeing the new alert manager structure marshal/unmarshal errors while running the test because of the edits it did. 

Do you think the old structure was working but the test was failing due to something else?

Reply:
The old test was failing due to changes in the alertmanager.yaml file and using the load function from config couldn't handle the format.